dropwizard-jooq/src/main/java/com/bendb/dropwizard/jooq/jersey/DataSourceParam.java

```diff
@@ -0,0 +1,10 @@
+package com.bendb.dropwizard.jooq.jersey;
+
+import java.lang.annotation.*;
+
+@Documented
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ElementType.PARAMETER})
+public @interface DataSourceParam {

```

Comment:
The name DataSourceParam is too different from Context. Maybe rename it to VariableContext to indicate that it is the same type of thing as the Context?